website/src/components/CardArticle/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
+
+import './styles.css';
+
+export interface CardArticleProps {
+  img?: string;
+  imgAlt?: string;
+  title?: string;
+  article?: string;
+  stringDate?:string;
+  authors?: string;
+}
+
+export const CardArticle: React.FC<CardArticleProps> = ({
+  img, // images must be squares
+  imgAlt = '',
+  stringDate = "",
+  title,
+  article,
+  authors,
+  children
+}) => {
+  const regex = /\//g;
+  const d_m_years = stringDate.split(regex);
```

Comment:
Why not just `"2022/03/01".split("/")`  ?

> Some people, when confronted with a problem, think "I know, I'll use regular expressions." Now they have two problems.

You can read a bit more context of the quote here : https://blog.codinghorror.com/regular-expressions-now-you-have-two-problems/ :grin:.
I'd say don't use regex when you can do simpler code, only use regex when really needed.